test/README.md

```diff
@@ -0,0 +1,105 @@
+# Tests
+
+###abort
+
+Tests for when the `--abort-on-uncaught-exception` flag is used.
+
+| Runs on CI |
+|:----------:|
+| No         |
+
+###addons 
+
+Tests for [addon](https://nodejs.org/api/addons.html) functionality.
+
+| Runs on CI |
+|:----------:|
+| Yes        |
+
+###cctest 
+
+C++ test that is run as part of the build process.
+
+| Runs on CI |
+|:----------:|
+| Yes        |
+
+###debugger
+
+Tests for [debugger](https://nodejs.org/api/debugger.html) functionality.
+
+| Runs on CI |
+|:----------:|
+| No         |
+ 
+###disabled
+
+Tests that have been disabled from running for various reasons.
+
+| Runs on CI |
+|:----------:|
+| No         |
+ 
+###fixtures
+
+Test fixtures used in various tests throughout the test suite.
+
+###gc
+
+Tests for garbage collection.
+
+| Runs on CI |
+|:----------:|
+| No         |
+ 
+###internet
+
+Tests for networking related modules ([DNS](https://nodejs.org/api/dns.html), [HTTP](https://nodejs.org/api/http.html), [Net](https://nodejs.org/api/net.html), [TLS](https://nodejs.org/api/tls.html)).
```

Comment:
Maybe give more details about how these actually make outbound connections. There are tests for all of those modules in both `/parallel` and `/sequential`. None of those actually go outside the local network though.